.circleci/ci/src/jobs/job-perf-lint-build.ts

```diff
@@ -0,0 +1,40 @@
+import { commands, Config, Job, reusable } from '@circleci/circleci-config-sdk';
+import { Command } from '@circleci/circleci-config-sdk/dist/src/lib/Components/Commands/exports/Command';
+import { NodeLtsExecutor } from '../executors';
+import { NotifyOnFailureCommand, WebuiInstallCommand } from '../commands';
+
+export class PerfLintBuildJob {
```

Comment:
Should this be in the "frontend" folder as well?

Reply:
and one of the run command which "builds" instead of "tests"